packages/base/Badge/src/Badge/Badge.tsx

```diff
@@ -49,25 +45,37 @@ export const Badge = forwardRef<HTMLDivElement, Props>(function Badge(
   },
   ref
 ) {
-  const classes = useStyles()
-
   const hasChildren = Children.count(children) > 0
 
   return (
     <MuiBadge
       ref={ref}
-      style={style}
       data-testid={testId}
       badgeContent={content}
       max={max || thresholds[size]}
-      className={className}
       showZero
-      classes={{
-        badge: cx(classes.root, classes[variant], classes[size], {
-          [classes.static]: !hasChildren,
-        }),
+      slotProps={{
+        root: {
+          className: twMerge(
+            `inline-flex flex-shrink-0 content-middle flex-nowrap justify-normal 
+            text-[10px] font-semibold leading-3 align-middle text-graphite-700 
+            relative top-0 right-0`,
```

Comment:
```suggestion
            top-0 right-0`,
```
we already have relative set below